arch/AArch64/AArch64Module.h

```diff
@@ -0,0 +1,13 @@
+/* Capstone Disassembly Engine */
+/* By Nguyen Anh Quynh <aquynh@gmail.com>, 2018 */
```

Comment:
now could you put your name here (and all other .h files), instead of mine?

Reply:
Implemented. I should really learn my name and email at this point...  :grinning: